roles/openshift_hosted/tasks/registry/secure.yml

```diff
@@ -25,7 +25,8 @@
     signer_serial: "{{ openshift_master_config_dir }}/ca.serial.txt"
     hostnames:
     - "{{ docker_registry_service_ip.results.clusterip }}"
-    - docker-registry.default.svc.cluster.local
+    - docker-registry.default.svc
+    - "docker-registry.default.svc.{{ openshift.common.dns_domain }}"
```

Comment:
This will change in the future (post 3.6 at the earliest), but right now the service is always named 'docker-registry' and in the 'default' namespace.